indico/worker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:2.7-buster
+FROM python:3
```

Comment:
I'd be explicit about `3.9` (or 3.10 once indico 3.2 is released)

Reply:
The only downside vs. the latest is that we have to keep manually updating it.

Reply:
yeah, but indico <3.1 won't work on 3.10, and indico <3.3 won't work on 3.11